am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -69,22 +71,29 @@ public void grantExplicitResourceAccess(ExplicitAccessGrant explicitAccessGrant)
             throw new IllegalArgumentException("At least one permission per attribute is required");
         }
 
+        if (explicitAccessGrant.getAccessorIds().stream()
```

Comment:
Throw away code :P I am implementing validations in a different way.

Reply:
👍